pycqed/measurement/waveform_control/sequence.py

```diff
@@ -132,11 +140,186 @@ def repeat_ro(self, pulse_name, operation_dict):
         """
         return self.repeat(pulse_name, operation_dict,
                            (self.n_acq_elements(), 1))
+
+
+    @staticmethod
```

Comment:
why are the 2 new functions staticmethods of the class? shouldn't they rather be just plain functions in the module?

Reply:
Yes, good question, I also thought about that. They could be plain functions in the module. I just thought it would avoid loading the sequence module everytime we deal with sequences (e.g. in mqm where we get sequence objects back but typically do not construct sequence in, so the module is not loaded). With static functions, we can just access them from the returned sequence objects. 
Is this a sufficent use-case to have them part of the object? @chellings Otherwise, I'll move them

Reply:
OK, then keep them as static